<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gsmViewer</class>
 <widget class="QWidget" name="gsmViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gsmViewer</string>
  </property>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btExit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCmd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>信号强度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Lac</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLac">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCellid">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btBase">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Base</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>run</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请求路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leRequest">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>prefix</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePrefix">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lon</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLon">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>lat</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLat">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btGetUrl">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>91</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Url</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btGetPos">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>81</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Pos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCoor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>最终经纬</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSource">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>600</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>刺客信条</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
